sdk/formrecognizer/Azure.AI.FormRecognizer/src/CustomFormModelInfo.cs

```diff
@@ -6,6 +6,8 @@
 namespace Azure.AI.FormRecognizer.Training
 {
     /// <summary>
+    /// Describes a model trained in a Cognitive Services Account and its training
+    /// status.
```

Comment:
There are two statuses around models -- one is the model status which tells where in the model lifecycle the model is -- either it's training or ready, or invalid, and later "copying" will be added to this enum.  There's also the training status which represents the _outcome_ of training - whether it succeeded, partially succeeded, or failed.  This is a bit of an awkward delineation - let's try to be consistent in how we call things so we don't add to the potential confusion around this.

Reply:
Does
```
Describes a model trained in a Cognitive Services Account and its status.
```
sound good? Or would you prefer a detailed description?

Reply:
That's great :)